api/server.py

```diff
@@ -576,7 +576,10 @@ def user_delete_user(id):
         endpoint = f"https://{AUTH0_DOMAIN}/api/v2/users/" + id
         headers = {"Authorization": f"Bearer {get_api_token()}"}
         response = requests.delete(endpoint, headers=headers)
-        return f"{response.status_code}"
+        if response.headers.get("Content-Type") == "application/json":
+            return jsonify(response.json()), response.status_code
+        else:
+            return response.text, response.status_code
```

Comment:
TIL about the [204 status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204)—which Auth0 sends when a user is successfully deleted.

It's probably safe to assume that the error modes send JSON content, but since the `204` status does not I went ahead and built the frontend to handle text content and not bother with JSON.